tensorflow/python/keras/layers/advanced_activations_test.py

```diff
@@ -88,6 +88,30 @@ def test_relu_with_invalid_arg(self):
             kwargs={'negative_slope': -2},
             input_shape=(2, 3, 4))
 
+  def test_rrelu_with_invalid_arg(self):
```

Comment:
This might needs testing for main case, i.e., create a RRelu and check the output in some way.

Reply:
Added one more test cases as requested by you and found a method mentioned in the spec to test it.This follows similar to the dropout method.